source/faq/faq.rst

```diff
@@ -259,95 +302,95 @@ Data Flows
 What is data flow?
 ^^^^^^^^^^^^^^^^^^
 
-It’s a processing flow for income messages/data of a device. With a flow you can dynamically 
-analyse each new message in order to apply validations, infer information, and trigger 
-actions or notifications. 
+It’s a processing flow for income messages/data of a device. With a flow you
+can dynamically analyse each new message in order to apply validations, infer
+information, and trigger actions or notifications.
 
 The data flow UI… really looks like node-RED. Are they related in some way?
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-It’s based on the Node-RED frontend, but uses its own engine to process the messages. 
-If you’re familiar with Node-Red, you won’t have any difficult to use it.
+It’s based on the Node-RED frontend, but uses its own engine to process the
+messages. If you’re familiar with Node-Red, you won’t have any difficult to use
```

Comment:
either:
> you won’t have any **difficulty** to use
or
> **it** won’t **be** difficult to use

I, particularly, like the second better.

Reply:
Ok